keras_retinanet/preprocessing/generator.py

```diff
@@ -244,3 +244,25 @@ def next(self):
             self.group_index = (self.group_index + 1) % len(self.groups)
 
         return self.compute_input_output(group)
+
+    # When left unspecified, steps_per_epoch takes on the value of len(generator)
+    # https://keras.io/models/sequential/#fit_generator
+    # This allows the keras_retinanet.preprocessing.generator.Generator object to be callable by len
+    def __len__(self):
+        return self.size()
+
+    def __next__(self):
```

Comment:
This function is already defined ?
https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/preprocessing/generator.py#L234-L235